kolibri/tasks/api.py

```diff
@@ -0,0 +1,67 @@
+import importlib
+from celery import Celery
+from django.core.management import get_commands, call_command
+
+import logging as logger
+
+logging = logger.getLogger(__name__)
```

Comment:
I had this way of using `getLogger` blow up on me on the weekend, in the other place it was used -- when it tried to actually use it, it said a logger of that name couldn't be found or something of the sort. Are we sure this is correct usage?